src/PageLayout/interaction.stories.tsx

```diff
@@ -0,0 +1,399 @@
+import {Meta} from '@storybook/react'
+import {StickyPane} from './PageLayout.stories'
+import {within} from '@storybook/testing-library'
+import {expect} from '@storybook/jest'
+
+const meta: Meta = {
```

Comment:
Do you think we can move this meta data out of the interaction stories and can be shared? As PageLayout.stories.tsx using the same data.